### ISSUE TITLE
exchange token redistribution

### ISSUE THREAD

**Issue by marcwoods109** [safemoonprotocol/Safemoon.sol#35]
I have safemoon on trust wallet and mxc exchange, im not getting any redistributed tokens on any exchange.

Anyonr know when they get done

Thanks

**Comment by sambeaCA** [comment]
If you deposited your Safemoon on an exchange that holds your token for you no redistribution will be given. "Not your keys, not your token". Keep them in your wallet and not on exchange.

**Comment by CryptoK91** [comment]
That is not true. The Exchanges uses different set ups for giving it out like BitMart for example do it like this.

SAFEMOON Transaction Fees Explanation: SAFEMOON token transactions charge extra 10% trading fees on top of BitMart Fees. This extra 10% fees include: 5% will be returned to the Developers to be added back to the liquidity pool for SAFEMOON tokens, and another 5% will be directly redistributed to all existing SAFEMOON token holders on BitMart. BitMart will take daily snapshots of users’ SAFEMOON transactions and distribute monthly. Distributions will be completed before the 10th of each month.

Read the details of the SafeMoon exchanges you are on. Im sure they have a similar way.

**Comment by sambeaCA** [comment]
Do you have verifiable proof? Show one from April 10th.
